test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DisasterRecovery/BackupServiceTests.cs

```diff
@@ -232,6 +232,44 @@ public void ScriptBackupWithAdvancedOptionsTest()
             testDb.Cleanup();
         }
 
+        /// <summary>
+        /// Test the correct script generation for different backup action types
+        /// </summary>
+        [Fact]
+        public void ScriptBackupWithDifferentActionTypesTest()
+        {
+            string databaseName = "testbackup_" + new Random().Next(10000000, 99999999);
```

Comment:
Suggest giving this a more descriptive name (SqlToolsService_TestBackup_**** or something) so it's really easy to identify where it's coming from

Reply:
Don't want to elongate name unless required. there aren't any other backup tests which do this - so seems like backup should do.

Reply:
We really shouldn't be concerned about length here. And my issue is that often I'll go on a test server and see a bunch of databases and have no idea where they're from. TestBackup is not descriptive enough to know that it's coming from the SqlToolsService tests. Being able to tell immediately where they're from will help save time if something goes wrong (like failing to clean up DBs)

Reply:
sure, will change